src/cli/deploy/index.js

```diff
@@ -83,24 +88,61 @@ module.exports = command(
     // 4b) Deploy
 
     for (let target of targets) {
-      const { publicPath, type } = target;
-      info(MESSAGES.deploy(target));
+      const { publicPath, type, to } = target;
+      let shouldOverwrite = false;
+
+      // If not ftp don't worry about checking external directory
+      if (type === 'ftp') {
+        const [checkErr] = await wrap(deploymentExists(to));
+
+        if (checkErr) {
+          if (checkErr.code === 550) {
+            // Directory doesn't exist. This is actually good though. OK to write.
+            shouldOverwrite = true;
+          } else {
+            error('An FTP error ocurred');
+          }
+        } else {
+          warn('Destination directory exists? OK to overwrite?');
+
+          const overwriteSelection = (
+            await cliSelect({
+              defaultValue: 0,
```

Comment:
Not sure how `cliSelect` config works, but it seems like 'Yes' might be the default here? 'No' would be a safer choice.

Reply:
Yep, that was a deliberate choice as most of the time you'll want to do the overwrite so it's just a single keypress confirmation, but absolutely a No default would be safer. Personally, I use `aunty build && aunty deploy` quite a bit to make latest working changes to interactives available to others while in the dev process and then proxy my local dev server using "Resource Override" in the browser. I can default it to No though and we'll check how it feels with the extra downarrow keypress.